edx/edx-app-ios
# iOS - Add a third tab to the discovery view named 'degrees'

### Description

[LEARNER-6918](https://openedx.atlassian.net/browse/LEARNER-6918)

In this PR i added the third tab name "degree" in discovery view.

### How to test this PR

Add the following configurations under the "DISCOVERY" object to enable degree
```
DEGREE:
        TYPE: 'webview'
        WEBVIEW:
            BASE_URL: 'https://webview.edx.org/course?type=Masters'
            DETAIL_TEMPLATE: 'https://webview.edx.org/{path_id}'
            SEARCH_ENABLED: true
```


- If you are not logged in, tap on the search courses, the third tab degree should be there.
- If you are logged in, switch the discovery tab view, there should be third tab degree there.
